cmd/neofs-adm/internal/modules/config/root.go

```diff
@@ -0,0 +1,29 @@
+package config
+
+import (
+	"github.com/spf13/cobra"
+)
+
+const configPathFlag = "path"
+
+var (
+	// RootCmd is a root command of config section.
+	RootCmd = &cobra.Command{
```

Comment:
is it ok to name both cobra command in that cli as "root"?

Reply:
It is. They are stored in different packages for this purpose.